Software/Frontend/app/src/main/java/hr/foi/air/baufind/ui/screens/RegistrationScreen/RegistrationScreen.kt

```diff
@@ -0,0 +1,203 @@
+// ui/screens/RegistrationScreen.kt
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import hr.foi.air.baufind.R
+import hr.foi.air.baufind.ui.components.PrimaryButton
+import hr.foi.air.baufind.ui.components.PrimaryTextField
+
+@Composable
+fun RegistrationScreen() {
+    var name by remember { mutableStateOf("") }
+    var email by remember { mutableStateOf("") }
+    var phone by remember { mutableStateOf("") }
+    var address by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+    var confirmPassword by remember { mutableStateOf("") }
+    var emailError by remember { mutableStateOf("") }
+    var nameError by remember { mutableStateOf("") }
+    var phoneError by remember { mutableStateOf("") }
+    var addressError by remember { mutableStateOf("") }
+    var passwordError by remember { mutableStateOf("") }
+    var confirmPasswordError by remember { mutableStateOf("") }
+
+    fun validateInputs(): Boolean {
+        var valid = true
```

Comment:
Validaciju možemo ostaviti, no kasnije ćemo glavni dio validacije prebaciti na backend projekt, ovdje će se provjeravati samo je li npr. sve što treba biti upisano, upisano.